### DISCUSSION TITLE
Website : confirmation mails are going strait into spam

### DISCUSSION THREAD

**Discussion by freeeflyer** [gusmanb/logicanalyzer#169] [General]
Hi. 
As the title says I was waiting for the confirmation mail and I found it in the spam

For your information, here is the report from spamassassin:
(some of those are on my side, 0.0 scores can be ignored)

X-Spam-Report:
    * 0.0 RCVD_IN_ZEN_BLOCKED_OPENDNS RBL: ADMINISTRATOR NOTICE: The query to
    * zen.spamhaus.org was blocked due to usage of an open resolver. See
    * https://www.spamhaus.org/returnc/pub/
    * [149.72.120.130 listed in zen.spamhaus.org]
    * 0.0 RCVD_IN_DNSWL_BLOCKED RBL: ADMINISTRATOR NOTICE: The query to DNSWL
    * was blocked. See
    * http://wiki.apache.org/spamassassin/DnsBlocklists#dnsbl-block for
    * more information.
    * [149.72.120.130 listed in list.dnswl.org]
    * 0.0 DKIM_ADSP_CUSTOM_MED No valid author signature, adsp_override is
    * CUSTOM_MED
    * 1.2 RCVD_IN_BL_SPAMCOP_NET RBL: Relais =?UTF-8?Q?list=E9?= dans
    * http://spamcop.net/bl.shtml
    * [Blocked - see <https://www.spamcop.net/bl.shtml?149.72.120.130>]
    * -1.0 RCVD_IN_MSPIKE_H2 RBL: Average reputation (+2)
    * [149.72.120.130 listed in wl.mailspike.net]
    * 0.0 SPF_HELO_NONE SPF: HELO does not publish an SPF Record
    * 0.1 DKIM_SIGNED Message has a DKIM or DK signature, not necessarily
    * valid
    * 0.1 DKIM_INVALID DKIM or DK signature exists, but is not valid
    * 0.0 FREEMAIL_FROM Sender email is commonly abused enduser mail provider
    * [logicanalyzerorders(at)gmail.com]
    * 1.0 FORGED_GMAIL_RCVD 'From' gmail.com does not match 'Received' headers
    * -2.0 RCVD_IN_VALIDITY_SAFE RBL: Sender in Validity Safe - Contact
    * certification@validity.com
    * [Excessive Number of Queries | <https://knowledge.validity.com/hc/en-us/articles/20961730681243>]
    * 0.1 MIME_HTML_ONLY BODY: Le message =?UTF-8?Q?poss=E8de?= uniquement des parties MIME
    * text/html
    * 0.0 HTML_MESSAGE BODY: HTML inclus dans le message
    * 1.8 HTML_IMAGE_ONLY_08 BODY: HTML contient images avec 600 =?UTF-8?Q?=E0?= 800 octets
    * de texte
    * 0.6 HTML_MIME_NO_HTML_TAG Message en HTML seul, mais sans tags HTML
    * 1.2 NML_ADSP_CUSTOM_MED ADSP custom_med hit, and not from a mailing list
    * 0.9 DMARC_NONE DMARC none policy
    * 1.0 SPOOFED_FREEMAIL No description available.
    * 0.0 T_REMOTE_IMAGE Message contains an external image
    * 1.0 SPOOF_GMAIL_MID From Gmail but it doesn't seem to be...

Hope this helps.

Nicolas

**Comment by gusmanb** [comment]
Thanks a lot for the report, will try to change the emailer, I'm using sendgrid and seems that is doing some nasty things.

**Comment by gusmanb** [comment]
I have replaced sendgrid directly with gmail, hopefully this will eliminate the spam problems. If anyone still finds the issue please leave a message here.


Thanks!
